packages/rum/src/userActionCollection.ts

```diff
@@ -65,43 +32,51 @@ export interface AutoUserAction {
 
 export type UserAction = CustomUserAction | AutoUserAction
 
+export function stopCurrentUserAction() {
+  if (currentUserAction) {
+    currentUserAction.stop()
+  }
+}
+
 export function startUserActionCollection(lifeCycle: LifeCycle) {
+  let stopNewUserAction: { stop(): void }
+  addEventListener(DOM_EVENT.CLICK, processClick, { capture: true })
   function processClick(event: Event) {
     if (!(event.target instanceof Element)) {
       return
     }
-
-    newUserAction(lifeCycle, UserActionType.CLICK, getElementContent(event.target))
+    stopNewUserAction = newUserAction(lifeCycle, UserActionType.CLICK, getElementContent(event.target))
   }
 
-  addEventListener(DOM_EVENT.CLICK, processClick, { capture: true })
-
   return {
     stop() {
+      stopNewUserAction.stop()
```

Comment:
`stopNewUserAction` is not necessarily defined (ex: if no click occured)

Reply:
ok I initialized it too noop for this scenario